backend/tests/v5/unit/processors/teamspaces/projects/models/commons/tickets.groups.test.js

```diff
@@ -37,244 +38,342 @@ const ScenesModel = require(`${src}/models/scenes`);
 jest.mock('../../../../../../../../src/v5/services/filesManager');
 const FilesManager = require(`${src}/services/filesManager`);
 
-const testGetTicketGroupById = () => {
-	describe('Get ticket group by Id', () => {
-		const teamspace = generateRandomString();
-		const project = generateRandomString();
-		const model = generateRandomString();
-		const revId = generateRandomString();
-		const ticket = generateRandomString();
-		const group = generateRandomString();
-
-		test('[Normal group] should return the group found', async () => {
-			const expectedData = generateRandomObject();
-			GroupsModel.getGroupById.mockResolvedValueOnce(expectedData);
+const externalIdNamesToKeys = {
```

Comment:
you shoulnd't be declaring constants that is declared in  source here. use an exposed constant object from source.